zebra-chain/src/transaction.rs

```diff
@@ -109,7 +109,7 @@ pub enum Transaction {
         /// The sapling shielded data for this transaction, if any.
         sapling_shielded_data: Option<sapling::ShieldedData<sapling::PerSpendAnchor>>,
     },
-    /// A `version = 5` transaction, which supports `Sapling` and `Orchard`.
+    /// An NU5 transaction (`version = 5`), which supports Orchard and Sapling (but not Sprout).
```

Comment:
Transaction versions are not 1:1 with network upgrades, and may be used for far longer than the network upgrade they are introduced in, so I would avoid this description:

```suggestion
    /// A `version = 5` transaction , which supports Orchard, Sapling, and transparent, but not Sprout.
```

Reply:
Thanks, @dconnolly. There was also a similar discussion here https://github.com/ZcashFoundation/zebra/pull/3100#discussion_r757093858.

Reply:
The reason why I edited this comment was that I wanted to make it more consistent with the ones for other transactions.